### PR TITLE
Feature/improve decks

### PR DESCRIPTION
**Pull request by martlind**
- Minimized How to play
- New deck headers (2 versions; sharp and round)
- Transparent top bar feature

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -105,13 +105,13 @@ export default {
 .hero {
   position: relative;
   overflow: hidden;
-
   padding: 64px 0;
+  z-index: 0;
```
Comment (daankuhlmann25) [inline src/views/Home.vue:109]: I think default z-index is always 0 so this is not needed, but doesn't matter to much.

File: src/assets/less/style.less [hunk 2]
```diff
@@ -270,6 +275,43 @@ body {
...
+        var(--gameHeaderDecorationSharp) center,
+        linear-gradient(265deg, rgba(28, 26, 33, 0.5) 0%, rgba(43, 37, 7, 0.75) 100%);
+        background-size: cover;
+        background-position-y: 57%;
+      }
+    }
+    
+    .game.trivia {
+      
+      header.top {
```
Comment (daankuhlmann25) [inline src/assets/less/style.less:305]: We could merge this with .game.trivia above, if there is no further styling needed.
But thats only readability.

**Review by daankuhlmann25 — APPROVED** [review]

File: src/views/Home.vue (hunk 1 above)
Comment (martlind) [reply src/views/Home.vue:109]: i _think_ i started there but had to do this. Im probably doing something wrong. 😓

File: src/assets/less/style.less (hunk 2 above)
Comment (martlind) [reply src/assets/less/style.less:305]: background-size is the only one that they have in common? In what way would you merge them?

**Pull request merged**